services/movie.py

```diff
@@ -0,0 +1,38 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None
```

Comment:
Add Optional to annotation, because you're setting None as default. And fix return annotation, it will be Queryset

Reply:
Fix all cases with `= None` in args, add Optional